api/server/daemon.go

```diff
@@ -178,3 +181,23 @@ func (s *Server) getEvents(ctx context.Context, w http.ResponseWriter, r *http.R
 		}
 	}
 }
+
+func getLabels(d *daemon.Daemon, id string) map[string]string {
+	container, err := d.Get(id)
```

Comment:
That makes a lot of sense. I did get the feeling this was the wrong place for it, but didn't want to go moving things around without more context.  

I'll get more of this moved into Daemon.